packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -121,7 +138,10 @@ class WebView extends StatefulWidget {
     this.gestureRecognizers,
     this.onPageFinished,
     this.debuggingEnabled = false,
+    this.initialMediaPlaybackPolicy =
+        AutoMediaPlaybackPolicy.require_user_action_for_all_media_types,
```

Comment:
Just checking--this is already the default, right? This should be treated as a breaking change otherwise.

Reply:
yes